src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -608,11 +549,13 @@ protected override void Dispose(bool disposing)
             state.WindowsProxyUsePolicy = _windowsProxyUsePolicy;
             state.Proxy = _proxy;
             state.ServerCredentials = _serverCredentials;
+            state.DefaultProxyCredentials = _defaultProxyCredentials;
+            state.PreAuthenticate = _preAuthenticate;
 
             try
             {
                 Task.Factory.StartNew(
-                    s => ((RequestState)s).Handler.StartRequest(s),
+                    s => ((WinHttpRequestState)s).Handler.StartRequest(s),
```

Comment:
I'd mentioned this in an earlier review I believe, but StartNew won't throw exceptions in this case, or if it does, it's something not-specific to StartNew, like an OOM.  The try/catch around this isn't necessary.